src/hacbs/components/PipelineRunDetailsView/PipelineRunDetailsView.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { useK8sWatchResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { Bullseye, Spinner } from '@patternfly/react-core';
+import { NamespaceContext } from '../../../components/NamespacedPage/NamespacedPage';
+import { PipelineRunGroupVersionKind } from '../../models';
+import { PipelineRunKind } from '../../types';
+import { calculateDuration } from '../../utils/pipeline-utils';
+import './PipelineRunDetailsView.scss';
+
+export interface PipelineRunDetailsProps {
+  pipelineRunName: string;
+}
+
+export const PipelineRunsDetailsView: React.FC<PipelineRunDetailsProps> = ({ pipelineRunName }) => {
+  const { namespace } = React.useContext(NamespaceContext);
+
+  const [pipelineRun, loaded] = useK8sWatchResource<PipelineRunKind>({
+    groupVersionKind: PipelineRunGroupVersionKind,
+    name: pipelineRunName,
+    namespace,
+  });
+
+  if (!loaded) {
+    return (
+      <Bullseye>
+        <Spinner />
+      </Bullseye>
+    );
+  }
+
+  const duration = calculateDuration(
+    typeof pipelineRun?.status?.startTime === 'string' ? pipelineRun?.status?.startTime : '',
+    typeof pipelineRun?.status?.completionTime === 'string'
+      ? pipelineRun?.status?.completionTime
+      : '',
+  );
+
+  return (
+    <>
+      <div className="co-m-pane__body odc-pipeline-run-details">
+        <dt className="hacbs-dt">Name</dt>
+        <dd className="hacbs-dd">{pipelineRun.metadata.name}</dd>
+
+        <dt className="hacbs-dt">Namespace</dt>
+        <dd className="hacbs-dd">{pipelineRun.metadata.namespace}</dd>
+
+        <dt className="hacbs-dt">Labels</dt>
+        <dd className="hacbs-dd">{'---'}</dd>
+
+        <dt className="hacbs-dt">Created at</dt>
+        <dd className="hacbs-dd">{'---'}</dd>
+
+        <dt className="hacbs-dt">Owner</dt>
+        <dd className="hacbs-dd">{'---'}</dd>
+
+        <dt className="hacbs-dt">Duration:</dt>
+        <dd className="hacbs-dd">{duration}</dd>
+
+        <dt className="hacbs-dt">Status</dt>
+        <dd className="hacbs-dd">
+          {pipelineRun.status.conditions[0].status === 'False' ? 'Failed' : 'Succeeded'}
+        </dd>
+
+        <dt className="hacbs-dt">Message</dt>
+        <dd className="hacbs-dd"> {pipelineRun.status.conditions[0].message}</dd>
```

Comment:
Patternfly has description list components https://patternfly-react.surge.sh/components/description-list#examples, we could use this instead to avoid custom styles for dt & dd tags. wdyt?